is_nodata.json

```diff
@@ -1,7 +1,7 @@
 {
     "id": "is_nodata",
     "summary": "Value is not a no-data value",
-    "description": "Checks whether the specified data is a missing data, i.e. equals to any of the no-data values / `null`.",
```

Comment:
add that "NaN" is not considered no-data (so return false)?

Reply:
Done.